GeneticInheritanceGraphLibrary/tables.py

```diff
@@ -674,6 +678,102 @@ def random_match_pos(self, rng):
                             return self.MRCApos(u[0] - loc - 1, v[0] - loc - 1, False)
                 loc -= x[1] - x[0]
 
+    def plot(
```

Comment:
For the moment, can we call this `_plot(...)` (with a leading underscore), to indicate that it's not really "public" yet?